packages/escape-html/src/index.js

```diff
@@ -64,7 +75,7 @@ export function escapeLessThan( value ) {
  * @return {string} Escaped attribute value.
  */
 export function escapeAttribute( value ) {
-	return escapeQuotationMark( escapeAmpersand( value ) );
+	return escapeGreaterThan( escapeQuotationMark( escapeAmpersand( value ) ) );
```

Comment:
Could use a comment explaining why it diverges from HTML spec.

Reply:
Ah yes, I think it got lost in the rebase. Added!